draft-ietf-quic-transport.md

```diff
@@ -1483,10 +1483,13 @@ subsequent sections.
 ## STREAM Frame {#frame-stream}
 
 STREAM frames implicitly create a stream and carry stream data. The type byte
-for a STREAM frame contains embedded flags, and is formatted as `11FSSOOD`.
+for a STREAM frame contains embedded flags, and is formatted as `1UFSSOOD`.
 These bits are parsed as follows:
 
-* The first two bits must be set to 11, indicating that this is a STREAM frame.
+* The first bit must be set to 1, indicating that this is a STREAM frame.
+
+* 'U' is the UNI bit, which is used to indicate this is a unidirectional
+  stream, and no stream frames may be sent in response.
```

Comment:
I think you have a different mental model of how this might work, or possibly you're commenting that I'm not solving an existing problem you think should be solved?

I'm not familiar with the WebRTC structure, but presumably for WebRTC each side would send RTP on unidirectional streams created by the appropriate side(odd for initiator and even for the other).

For IMAP, if the server speaks first, then I would expect it to also create an even stream ID and send communications on that.